app.lua

```diff
@@ -85,7 +85,7 @@ domain_allowed['studio.edx.org'] = true
 domain_allowed['edge.edx.org'] = true
 -- Development
 domain_allowed['romagosa.work'] = true
-domain_allowed['localhost:8080'] = true
+domain_allowed['localhost'] = true
```

Comment:
hmmm, are you sure about this change? I think you need to specify the port. I'm not sure `localhost` will cover `localhost:8080` or any other port.

(I use this setting when developing the social site)

Reply:
Yes, because I'm normalizing the URL. Somewhat stupidly, CORS treats `example.com` and `example.com:80` as different origins, even though they are the same port. 

I definitely use it too. Filtering the port is helpful because I don't always run things on 8080 either.

Reply:
https://github.com/bromagosa/snapCloud/pull/69/files#diff-d5c255fc66c6740437720ea92cd0815fR106